articles/other/other-how-delete-sso-user.md

```diff
@@ -0,0 +1,190 @@
+     ```
+
+    This returns a single sign on token to use for authenticating calls to the IDAM API. For example:
+
+    ``` json
+    {
+     "access_token": "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJUU0F1WERTWUt0SGVEWTI5ZHlPVFZVeHZUMWNhYmUyNUpEVVZzajhVRUhrIn0.eyJqdGkiOiI2YWJmYzkw...",
+     "expires_in": 300,
+     "refresh_expires_in": 1800,
+     "refresh_token": "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJUU0F1WERTWUt0SGVEWTI5ZHlPVFZVeHZUMWNhYmUyNUpEVVZzajhVRUhrIn0.eyJqdGkiOiIyMDkyNjA0...",
+     "token_type": "bearer",
+     "not-before-policy": 0,
+     "session_state": "342ab476-d286-41b9-8b3a-1a236cceefc1"
+    }
+    ```
+
+    For convenience, the token is saved to the `token` variable in your current shell session. Use this variable in the `Authorization: Bearer` header for all further calls to the IDAM API.
+
+2. List users and note the ID of the user you want to delete:
+
+    ``` bash
+    curl -s -H "Authorization: Bearer $token" https://idam.ukcloud.com/v1/users | jq -r '.[] | .id + " " + .username'
+    ```
+
+    This returns a list of SSO users:
+
+    ``` json
+    [
+      {
+        "id": "0744f665-2d7a-483c-adb3-72b0effe2356",
+        "createdTimestamp": 1544026027038,
+        "username": "acc1admin",
+        "enabled": true,
+        "totp": false,
+        "emailVerified": false,
+        "disableableCredentialTypes": [
+          "password"
+        ],
+        "requiredActions": [],
+        "notBefore": 0,
+        "access": {
+          "manageGroupMembership": true,
+          "view": true,
+          "mapRoles": true,
+          "impersonate": false,
+          "manage": true
+        }
+      },
+      {
+        "id": "1d9e52e6-f833-47b9-9cfa-be727510c910",
+        "createdTimestamp": 1544026027145,
+        "username": "acc1usr1",
+        "enabled": true,
+        "totp": false,
+        "emailVerified": false,
+        "disableableCredentialTypes": [
+          "password"
+        ],
+        "requiredActions": [],
+        "notBefore": 0,
+        "access": {
+          "manageGroupMembership": true,
+          "view": true,
+          "mapRoles": true,
+          "impersonate": false,
+          "manage": true
+        }
+      },
+      {
+        "id": "a089698b-8f32-4a32-ab71-790b015aa662",
+        "createdTimestamp": 1544026027295,
+        "username": "acc1usr2",
+        "enabled": true,
+        "totp": false,
+        "emailVerified": false,
+        "disableableCredentialTypes": [
+          "password"
+        ],
+        "requiredActions": [],
+        "notBefore": 0,
+        "access": {
+          "manageGroupMembership": true,
+          "view": true,
+          "mapRoles": true,
+          "impersonate": false,
+          "manage": true
+        }
+      },
+    ]
+    ```
+
+3. Find the user that you want to delete and make a note of that user's ID. For example, to delete the user `acc1usr2`, you would need the ID `a089698b-8f32-4a32-ab71-790b015aa662`.
```

Comment:
Perhaps it could be suggested to keep this in a variable for reuse if the user should be removed from multiple groups, e.g.

```
acc1usr2=a089698b-8f32-4a32-ab71-790b015aa662
curl -X DELETE -H "Authorization: Bearer $token" https://idam.ukcloud.com/v1/users/$acc1usr2/groups/008d3512-54f9-439e-800a-7b0b5ef76438
```

Reply:
Fixed